README.md

```diff
@@ -124,7 +124,7 @@ in various programming languages.
   by Ningning Xie and Daan Leijen  
   ([pdf](https://www.microsoft.com/en-us/research/uploads/prod/2020/07/effev.pdf))
 
-* **Effekt: Lightweight Effect Polymorphism for Handlers** (OOPSLA 2020)  
+* **Effekt: Lightweight Effect Polymorphism for Handlers** (conditionally accepted at OOPSLA 2020)  
```

Comment:
I suggest we either delete this entry or change it to "technical report", this is my preferred solution as it complements the software section, then we can update this entry once the paper has been fully accepted at OOPSLA.
```suggestion
* **Effekt: Lightweight Effect Polymorphism for Handlers** (Technical report 2020)  
```